JavaScript/chapter02/p07_intersection/avc278.js

```diff
@@ -0,0 +1,119 @@
+// Intersection: Given two (singly) linked lists, determine if the two lists intersect. Return the inter­secting node.
+// Note that the intersection is defined based on reference, not value. That is, if the kth node of the first linked
+// list is the exact same node (by reference) as the jth node of the second linked list, then they are intersecting.
+
+const assert = require("assert");
+const { LinkedListNode } = require("../../lib/avc278/linkedlist");
+
+/**
+ *
+ * @param   {LinkedListNode} A input linked list to check against
+ * @param   {LinkedListNode} B input linked list to check against
+ * @return  {any}              whether the input linked list is a palindrome
```

Comment:
the type bounds can be improved! (and the return type message too)

```suggestion
 * @return  {LinkedListNode|null}         whether the input linked lists intersect
```